create-devops-pat.azcli

```diff
@@ -0,0 +1,5 @@
+# This is a way to create an Azure DevOps personal access token using the Azure CLI and REST Endpoints.
+
+# The trick of this is that it uses the Azure CLI's access token to create an access token to the https://management.core.windows.net/ resource which also manages Azure DevOps instances.
+
+az rest --method post --uri "https://vssps.dev.azure.com/{your-org-name-here}/_apis/Tokens/Pats?api-version=6.1-preview" --resource "https://management.core.windows.net/" --body '{​​​​​​​ "displayName": "{your-pat-name-here}" }​​​​​​​' --headers "Content-Type"="application/json"
```

Comment:
love it

can you extract the variables so they are easy to find and edit?  

Can you also include a sample to get the PAT by name?

Can you file an issue here to provide this as part of the az devops extension? https://github.com/Azure/azure-devops-cli-extension

Reply:
> can you extract the variables so they are easy to find and edit?
> 

Good call out - does it matter which shell? I can just do a bash example maybe - inside .azcli file?

> Can you also include a sample to get the PAT by name?
> 

I can, but in Azure DevOps itself you only get the metadata of the PAT on a GET not the actual token itself. 

> Can you file an issue here to provide this as part of the az devops extension? https://github.com/Azure/azure-devops-cli-extension

Yes will do :smile:

Reply:
yeah, right in the azcli file, bash is fine

okay, if there's no way to get the actual PAT, then maybe it doesn't matter.  you can exclude if you want

Reply:
✅ Ok updated-  thank you 😄